pkg/network/encoding/encoding_test.go

```diff
@@ -23,6 +24,14 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type connTag = uint64
+
+// ConnTag constant must be the same for all platform
+const (
+	tagGnuTLS  connTag = 1 // netebpf.GnuTLS
+	tagOpenSSL connTag = 2 // netebpf.OpenSSL
```

Comment:
Do you know why we aren't using the types from `pkg/network/ebpf` here?

Reply:
These are only defined for linux in that package. So either we move the tests that use them to be linux only or we define them here.

Reply:
Got it. Thanks!